packages/flutter/lib/src/gestures/eager.dart

```diff
@@ -16,7 +16,10 @@ class EagerGestureRecognizer extends OneSequenceGestureRecognizer {
   /// Create an eager gesture recognizer.
   ///
   /// {@macro flutter.gestures.GestureRecognizer.kind}
-  EagerGestureRecognizer({ PointerDeviceKind? kind }) : super(kind: kind);
+  EagerGestureRecognizer({
+    PointerDeviceKind? kind,
+    Set<PointerDeviceKind>? kindSet,
```

Comment:
This duplication in the API is kinda cumbersome...

Reply:
What do you think? Do we deprecate `kind`?

Reply:
You could have the member variable be a Set (I'd also call it "kinds") and have the default constructor stay the same, wrapping a non-null `kind` in a set as needed -- and then providing a new constructor that took the set (e.g. something like `GestureRecognizer.multiDevice()`).

Reply:
This would work only on the customer level of API. And since the proposal is to make `GestureDetector` receive more than one `pointerDeviceKind`, this kind of misses the point.